bundle/language_en.properties

```
@@ -665,6 +665,7 @@ angal.hospital.visitduration.txt
 angal.hospital.visitendhour.txt                                                                        = Visit end hour
 angal.hospital.visitstarthour.txt                                                                      = Visit start hour
 angal.inventory.allproduct.txt                                                                         = All products
+angal.inventory.askforrealquantityempty                                                                = Do you want to print without the real quantity?
```

```suggestion
angal.inventory.askforrealquantityempty.msg                                                            = Do you want to print without the real quantity?
```
please use project suffixes; questions/message/etc (things in dialogs) should end in `.msg`